src/CurrencyInput/index.jsx

```diff
@@ -0,0 +1,183 @@
+/*
+ * Copyright (c) 2017-2019 dunnhumby Germany GmbH.
+ * All rights reserved.
+ *
+ * This source code is licensed under the MIT license found in the LICENSE file
+ * in the root directory of this source tree.
+ *
+ */
+
```

Comment:
```suggestion

/* global navigator */

```
Just to prevent linter from complaining `'navigator' is not defined`